ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Node/NodePageVulnerabilities.tsx

```diff
@@ -108,15 +119,15 @@ function NodePageVulnerabilities({ nodeId }: NodePageVulnerabilitiesProps) {
                                 page={page}
                                 onSetPage={(_, newPage) => setPage(newPage)}
                                 onPerPageSelect={(_, newPerPage) => {
+                                    setPerPage(newPerPage);
                                     if (nodeCount < (page - 1) * newPerPage) {
-                                        setPage(1);
+                                        setPage(1, 'replace');
```

Comment:
Something I missed in recent reviews, is the `if` statement redundant?
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/hooks/useURLPagination.ts#L31

Reply:
Aha yes! I've been carrying this pattern along out of habit, but it looks like a better and more efficient fix went in [9 months ago](https://github.com/stackrox/stackrox/pull/7652).

I'll refactor these out for Node/Platform CVEs in a follow up cleanup PR.